container/health.go

```diff
@@ -19,12 +19,7 @@ func (s *Health) String() string {
 		return types.Unhealthy
 	}
 
-	switch s.Status {
-	case types.Starting:
-		return "health: starting"
```

Comment:
Also, it perhaps made more sense with the original PR when health was broken out into another column in `docker ps`. I'm open to going back, but I think it would be best to keep to 1 value.